library/tendrl_user.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/python
+'''
+module: test_user
+short_description: Changes 'usm_username' user password to _password_from_module_argument_
+via Tendrl API.
+description:
+    - Module reads from confile(module argument 'conf_path') 'usm_username',
+      'usm_password' and 'usm_api_url'.
+    - There must be installed tendrl-api on the machine where this is ran.
+    - Usage: ansible -i _inventory_ -m tendrl_user -a "conf_path='_usm.ini.path_'" localhost
+version_added: "1.0"
+author: "fbalak@redhat.com", "mkudlej@redhat.com"
+'''
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+module: test_user
+short_description: Changes 'usm_username' user password to _password_from_module_argument_
+via Tendrl API.
+description:
+    - Module reads from confile(module argument 'conf_path') 'usm_username', 
+      'usm_password' and 'usm_api_url'.
+    - There must be installed tendrl-api on the machine where this is ran.
+version_added: "1.0"
+    - Usage: ansible -i _inventory_ -m tendrl_user -a "conf_path='_usm.ini.path_'" localhost
```

Comment:
Usage should be [EXAMPLES block](https://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html#examples-block)